### PR TITLE
Release 4.0.3

### PR DESCRIPTION
**Pull request by FinalAngel**
Required for aldryn to tmp speed up the build /cc @vxsx @stefanfoulis 

the "dependencies" can be installed with ``npm install --production`` which will ignore devDependnecies

### CODE REVIEW

**Comment by vxsx** [comment]
:+1:

**Comment by stefanfoulis** [comment]
``gulp sass`` fails because ``"minimist": "^1.1.3",`` is not in the production dependencies. I'll try to fix it.

**Comment by stefanfoulis** [comment]
ok... so nothing really works. the ``gulpfile.js`` even requires protractor. @FinalAngel this is what I meant about us probably needing 2 gulpfiles, since the ``require`` still require the things.

**Comment by stefanfoulis** [comment]
and I saw you removed ``gulp watch``. We still need that for local sass compilation.

**Comment by vxsx** [comment]
we could move the tasks to separate files and require only ones that needed based on env variable / command argument

**Comment by FinalAngel** [comment]
@vxsx can you check this with @stefanfoulis today so we can get this working?

File: gulpfile.js [hunk 1]
```diff
@@ -69,166 +50,44 @@ var PROJECT_PATTERNS = {
...
-            proxy: '0.0.0.0:' + PORT,
-            port: PORT + 1,
-            ui: {
-                'port': PORT + 2
-            }
-        });
-    }, 1000);
-});
+gulp.task('bower', task('bower'));
+gulp.task('lint:javascript', task('lint/javascript'));
```
Comment (FinalAngel) [inline gulpfile.js]: we don't need to run linting on aldryn

Comment (vxsx) [inline gulpfile.js]: yes, that's for local

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**